own_poem

```diff
@@ -0,0 +1,4 @@
+Roses are red,
```

Comment:
Very nice poem.  Can you add a comma at the end of lines 2 and 3?

Reply:
merge

Reply:
Can you add a comment here after you make the requested changes (adding commas at the end of lines 2 and 3). That way we'll be notified once you're done.